src/scripts/main.js

```diff
@@ -1,3 +1,35 @@
 'use strict';
 
 // write code here
+
+const list = document.querySelector('ul');
+const items = [...list.children];
+
+function sortList(elements) {
+  [...elements]
+    .sort((a, b) => {
+      const salaryA = formattedSalary(a.getAttribute('data-salary'));
+      const salaryB = formattedSalary(b.getAttribute('data-salary'));
+
+      return salaryB - salaryA;
+    })
+    .forEach((person) => {
+      list.appendChild(person);
+    });
+}
+
+function formattedSalary(salary) {
+  return Number(salary.replace(/,/g, '.').replace(/[$]/g, ''));
+}
+
+function getEmployees(elements) {
+  return elements.map((person) => ({
+    name: person.textContent.trim(),
+    position: person.getAttribute('data-position'),
+    salary: `$` + formattedSalary(person.getAttribute('data-salary')).toFixed(3),
+    age: Number(person.getAttribute('data-age')),
+  }));
+}
+
+getEmployees(items);
```

Comment:
I was wondering what for is that, but I see that its in the requirements.

Reply:
Yup, I also don't know why should i do it, without using it.. but well..